### PR TITLE
Add mobile nav

### PR DESCRIPTION
**Pull request by elv-zenia**
Add mobile navigation menu

### CODE REVIEW

File: src/main/components/Header.js [hunk 1]
```diff
@@ -94,9 +95,49 @@ const Header = observer(() => {
             Discover Projects
           </Action>
         </nav>
-        <Action icon={MenuIcon} className="dark header__mobile-nav-button mobile" onClick={() => {}} />
+        <Action icon={MenuIcon} className="dark header__mobile-nav-button mobile" onClick={() => setShowMobileMenu(prevState => !prevState)} />
       </header>
 
+      <div className={`mobile-nav__menu ${showMobileMenu ? "" : "hidden"}`}>
```
Comment (elv-kevin) [inline src/main/components/Header.js:101]: Let's move the mobile menu to a different file, and don't actually render it unless it's open.

File: src/main/components/Header.js [hunk 2]
```diff
@@ -94,9 +95,49 @@ const Header = observer(() => {
...
+            <Action to="/creators-and-publishers">Creators & Publishers</Action>
+            <Action to="/content-fabric">Content Fabric</Action>
+            <Action to="/content-fabric/technology">Technology</Action>
+            <Action to="/content-fabric/blockchain">Blockchain</Action>
+            <Action to="/features/details">Features</Action>
+          </div>
+          <hr className="mobile-nav__menu-section-line" />
+
+          <div className="mobile-nav__menu-links mobile-nav__menu-section mobile-nav__menu-secondary-links">
+            <Action className="link" to="/features/pricing">Pricing</Action>
```
Comment (elv-kevin) [inline src/main/components/Header.js:125]: is the "link" class being used? if not, omit it, or just give these a class for the mobile-nav section (.mobile-nav__link)

**Review by elv-kevin — CHANGES_REQUESTED** [review]

File: src/main/components/MobileNav.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+const MobileNav = ({open, onOpenChange}) => {
+  if(!open) { return null; }
+
+  return (
+    <div className="mobile-nav__menu">
+      <Action icon={XIcon} className="dark mobile-nav__close-button mobile" onClick={() => onOpenChange(false)} />
+      <div className="mobile-nav__menu-content">
+        <div className="mobile-nav__menu-header mobile-nav__menu-section">
+          <Action className="mobile-nav__menu-header-item" icon={DiscoverIcon} alt="Projects" to="/about">
+            <span>Projects</span>
```
Comment (elv-kevin) [inline src/main/components/MobileNav.js:14]: is there a reason for having additional <span>s here?

Comment (elv-zenia) [reply src/main/components/MobileNav.js]: I was styling the font size but I moved that to the parent and removed the span. I think I had more css for that span before.

**Pull request merged**